public/app/plugins/datasource/elasticsearch/test-helpers/render.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ComponentProps, ReactNode } from 'react';
```

Comment:
If you decide to remove `DateHistogramSettingsEditor` test, this can be removed as well, as it is not used anywhere else.

Reply:
Good point! I was considering the idea of using this in other tests as well (instead of using `ElasticsearchProvider` every time). do you think I should remove it now and reintroduce it only once I decide to refactor them?

Reply:
Hmmm, I would personally say so. 🙂 But I am okay with keeping it as well, as long as you are going to refactor it soon-ish. Just so it doesn't end up as forgotten file.

Reply:
Damn, forgot about this and merged 🙏 
I'll make sure is not going to be forgotten